### PR TITLE
Advanced installations

### PR DESCRIPTION
**Pull request by krahulreddy**
For issue #1683:

* Added script to import multiple country extracts.
* Added script to update database setup using the above script.
* Added 'Advanced Installations' documentation.

### CODE REVIEW

File: docs/admin/Import-and-Update.md [hunk 1]
```diff
@@ -321,3 +321,5 @@ If you have imported multiple country extracts and want to keep them
 up-to-date, have a look at the script in
 [issue #60](https://github.com/openstreetmap/Nominatim/issues/60).
 
+**Update**: [Advanced installations section](Advanced-Installations.md) contains instructions 
+to set up and update multiple country extracts.
```
Comment (lonvia) [inline docs/admin/Import-and-Update.md:325]: Just replace the paragraph above with your new one.

File: utils/import_multiple_regions.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+# SET TO YOUR replication server URL:
+
+BASEURL="https://download.geofabrik.de"
+DOWNCOUNTRYPOSTFIX="-latest.osm.pbf"
+
+# End of configuration section
+# ******************************************************************************
+
+COMBINEFILES="osmium merge"
```
Comment (lonvia) [inline utils/import_multiple_regions.sh:49]: You probably should add a quick note in the documentation that osmium-tool is needed and where to get it from.

**Comment by krahulreddy** [comment]
Resolved issues with [5c56ea3](https://github.com/osm-search/Nominatim/pull/1758/commits/5c56ea31984fe043bc0efe73e616017cbabfc909).
I had to force push after making local git config changes. Sorry for the inconvenience caused by that.

**Comment by lonvia** [comment]
Thanks!

As a general rule: force pushes on your PR branches are perfectly fine with me. In fact, I very much prefer that PRs are rebased on master instead of merged with it. And this works of course only if you force-push afterwards.

**Pull request merged**